persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -60,12 +61,16 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor wit
 
     case ReadHmdaRawFile(persistenceId) =>
       publishEvent(ParsingStarted(submissionId))
+      var encounteredParsingErrors: Boolean = false
```

Comment:
Thanks for the feedback. Could you say more about why having `encounteredParsingErrors` is safer as a class-level variable (that's what I think you're suggesting; correct me if I'm wrong), rather than as a method-level variable (which it is now)? That info will help me understand how to use vars in the future.